index.js

```diff
@@ -71,75 +71,98 @@ const data = [
-        description.innerHTML = data[1]["description"];
-        image.src = "media/SnapshootPortfolio2.png"
+    secondproject.addEventListener("click", () => {
+    control.classList.add("project-details-overlay-outer"); 
+    popup.style.display = "block";
+    topic.innerHTML = data[1]["title"];
+    names.innerHTML = data[1]["name"];
+    position.innerHTML = data[1]["position"];
+    year.innerHTML = data[1]["year"];
+    description.innerHTML = data[1]["description"];
+    image.src = "media/SnapshootPortfolio2.png"
+
+})
+
+thirdproject.addEventListener("click", () => {
+    control.classList.add("project-details-overlay-outer");
+    popup.style.display = "block";
+    topic.innerHTML = data[2]["title"];
+    names.innerHTML = data[2]["name"]
+    position.innerHTML = data[2]["position"]
+    year.innerHTML = data[2]["year"]
+    description.innerHTML = data[2]["description"];
+    image.src = "media/picture3.png"
+
+})
+
+fourthproject.addEventListener("click", () => {
+    control.classList.add("project-details-overlay-outer");
+    popup.style.display = "block";
+    topic.innerHTML = data[3]["title"];
+    names.innerHTML = data[3]["name"]
+    position.innerHTML = data[3]["position"]
+    year.innerHTML = data[3]["year"]
+    description.innerHTML = data[3]["description"];
+    image.src = "media/SnapshootPortfolio4.png"
+
+})
+
+exit.addEventListener("click", () => {
+    control.classList.remove("project-details-overlay-outer");
+    popup.style.display = "none";
+
+})
  
-  })
-    
-    thirdproject.addEventListener("click", () => {
-        control.classList.add("project-details-overlay-outer");
-        popup.style.display = "block";
-        topic.innerHTML = data[2]["title"];
-        names.innerHTML = data[2]["name"]
-        position.innerHTML = data[2]["position"]
-        year.innerHTML = data[2]["year"]
-        description.innerHTML = data[2]["description"];
-        image.src = "media/picture3.png"
-
-    })
-
-    fourthproject.addEventListener("click", () => {
-        control.classList.add("project-details-overlay-outer");
-        popup.style.display = "block";
-        topic.innerHTML = data[3]["title"];
-        names.innerHTML = data[3]["name"]
-        position.innerHTML = data[3]["position"]
-        year.innerHTML = data[3]["year"]
-        description.innerHTML = data[3]["description"];
-        image.src = "media/SnapshootPortfolio4.png"
-
-    })
-
-    exit.addEventListener("click", () => {
-        control.classList.remove("project-details-overlay-outer");
-        popup.style.display = "none";
-
-  })
- 
\ No newline at end of file
+
+const form = document.getElementById("html-form");
+const email = document.getElementById("email");
+const message = document.getElementById("message");
+
+form.addEventListener("submit", (event) => {
+
+    if(email.value !== email.value.toLowerCase()){
+        event.preventDefault()
+        message.textContent = "EMAIL should be in lowercase";
```

Comment:
Kindly change the error message of the email input position to be near the submit button instead. As it's in the requirement of the project.
![image](https://user-images.githubusercontent.com/24703171/156887181-cafb66e3-7fdc-4f81-b428-fd7dcd8fc2a2.png)